src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -116,13 +111,42 @@ private static bool IsNamespaceName(NameSyntax name)
             }
         }
 
+        private void TryClassifyStaticSymbol(
+            ISymbol symbol,
+            Text.TextSpan span,
+            ArrayBuilder<ClassifiedSpan> result)
+        {
+            if (symbol?.IsStatic == true
+                && (!symbol.IsKind(SymbolKind.Field) || symbol.ContainingType?.IsEnumType() == false) // TODO: Since Enum members are always static is it useful to classify them as static?
+                && !symbol.IsKind(SymbolKind.Namespace)) // TODO: Since Namespace are always static is it useful to classify them as static?
+            {
+                result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
+            }
+        }
+
         private bool TryClassifySymbol(
             NameSyntax name,
             ISymbol symbol,
             SemanticModel semanticModel,
             CancellationToken cancellationToken,
             out ClassifiedSpan classifiedSpan)
         {
+            if (symbol is INamespaceSymbol namespaceSymbol
+                && name is IdentifierNameSyntax identifierNameSyntax)
```

Comment:
why do you need the IdentifierName check?

Reply:
The namespaceSymbol for Extern Alias will return true for IsGlobalNamespace. But I am specifically looking to ignore the GlobalKeyword from additional classification. I'll try to find a better approach and add a comment to clarify what is going on.